src/memex/views.py

```diff
@@ -158,6 +158,7 @@ def create(request):
     schema = schemas.CreateAnnotationSchema(request)
     appstruct = schema.validate(_json_payload(request))
     annotation = storage.create_annotation(request, appstruct)
+    request.db.flush()
```

Comment:
As discussed in Slack, the annotation had an ID rather accidentally because of how the code was structured (objects added to the session and later some other part of the code flushed the session). I've removed the flush from the view and added it explicitly as an API contract of `create_annotation` (906770ce782f910222f92aa46683bd7799906245).